src/Integration/IServiceProviderExtensions.cs

```diff
@@ -58,5 +59,12 @@ public static T GetMefService<T>(this IServiceProvider serviceProvider)
             // We don't want to throw in the case of a missing service (don't use GetService<T>)  
             return componentModel?.GetExtensions<T>().SingleOrDefault();
         }
+
+        [Conditional("DEBUG")]
+        internal static void AssertLocalServiceIsNotNull<T>(this T service)
```

Comment:
Defined a marker interface (ILocalService) which all the service interface are now inherit. Lazy<object> became Lazy<ILocalService>